File: Java/irrgarten/GameState.java
package irrgarten;

public class GameState {
    private String laberinth;
    private String players;
    private String monsters;
    private int currentPlayer;
    private boolean winner;
    private String log;

    public GameState(String laberinth, String players, String monsters, int currentPlayer, boolean winner, String log) {
        this.laberinth = laberinth;
        this.players = players;
        this.monsters = monsters;
        this.currentPlayer = currentPlayer;
        this.winner = winner;
        this.log = log;
    }

    public String getLabyrinth() {
        return laberinth;
    }

    public String getPlayers() {
        return players;
    }

    public String getMonsters() {
        return monsters;
    }

    public int getCurrentPlayer() {
        return currentPlayer;
    }

    public boolean isWinner() {
        return winner;
    }

    public String getLog() {
        return log;
    }
}

File: Ruby/Irrgarten/game_state.rb
class Game_state
    def initialize(labyrinth, players, monsters, current_player, winner, log)
        @labyrinth = labyrinth
        @players = players
        @monsters = monsters
        @current_player = current_player
        @winner = winner
        @log = log
    end

    def get_labyrinth()
        return @labyrinth
    end

    def get_players()   
        return @players
    end

    def get_monsters()
        return @monsters
    end

    def get_current_player()
        return @current_player
    end

    def get_winner()
        return @winner
    end

    def get_log()
        return @log
    end
end